<ui version="4.0" >
 <class>TestDialog</class>
 <widget class="QDialog" name="TestDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>그림</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>채도</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QSlider" name="horizontalSlider" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox" />
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QCheckBox" name="checkBox" >
         <property name="text" >
          <string>색반전하기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>미리보기</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="color_group" >
     <property name="title" >
      <string>색상값 지정하기</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QRadioButton" name="rgb_radio" >
        <property name="text" >
         <string>RGB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cmyk_radio" >
        <property name="text" >
         <string>CMYK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="yuv_radio" >
        <property name="text" >
         <string>YUV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="color_combo" >
        <item>
         <property name="text" >
          <string>RGB</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>CMYK</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>YUV</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>188</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>219</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>238</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>95</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>61</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel" >
     <x>115</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TestDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>130</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel" >
     <x>119</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TestDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>225</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel" >
     <x>272</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
